app/controllers/serials_controller.rb

```diff
@@ -1,6 +1,8 @@
 class SerialsController < ApplicationController
   def index
     @serials = current_site.serials.published.order(id: :desc).page(params[:page])
```

Comment:
In SQL specification, unordered query returns unsorted result possibly.
So we need specify order, if we need stable sorted result.